static/js/composebox_typeahead.js

```diff
@@ -343,10 +343,16 @@ export function tokenize_compose_str(s) {
 }
 
 export function broadcast_mentions() {
+    let wildcard_string = "";
+    if (compose_state.get_message_type() === "private") {
+        wildcard_string = $t({defaultMessage: "Notify recipients"});
+    } else {
+        wildcard_string = $t({defaultMessage: "Notify stream"});
+    }
     return ["all", "everyone", "stream"].map((mention, idx) => ({
         special_item_text: $t(
-            {defaultMessage: "{wildcard_mention_token} (Notify stream)"},
-            {wildcard_mention_token: mention},
+            {defaultMessage: "{wildcard_mention_token} ({wildcard_string})"},
+            {wildcard_mention_token: mention, wildcard_string},
```

Comment:
I don't think we need this block tagged for translation; there aren't any language-specific bits in `{wildcard_mention_token} ({wildcard_string})`, and it's likely to just confuse translators. I've converted that to a string literal.